openslides/core/signals.py

```diff
@@ -40,47 +40,48 @@ def setup_general_config(sender, **kwargs):
             required=False,
             max_length=100))
 
-    event_date = ConfigVariable(
-        name='event_date',
+    general_event_date = ConfigVariable(
+        name='general_event_date',
         default_value='',
         form_field=forms.CharField(
             widget=forms.TextInput(),
             label=ugettext_lazy('Event date'),
             required=False))
 
-    event_location = ConfigVariable(
-        name='event_location',
+    general_event_location = ConfigVariable(
+        name='general_event_location',
         default_value='',
         form_field=forms.CharField(
             widget=forms.TextInput(),
             label=ugettext_lazy('Event location'),
             required=False))
 
-    event_organizer = ConfigVariable(
-        name='event_organizer',
+    # TODO: Check whether this variable is ever used.
+    general_event_organizer = ConfigVariable(
+        name='general_event_organizer',
         default_value='',
         form_field=forms.CharField(
             widget=forms.TextInput(),
             label=ugettext_lazy('Event organizer'),
             required=False))
 
-    projector_enable_logo = ConfigVariable(
-        name='projector_enable_logo',
+    general_projector_enable_logo = ConfigVariable(
+        name='general_projector_enable_logo',
```

Comment:
I would prefer that we use a own prefix for all "projector" configs to group projector configs in one section. So my proposal: remove "general_" from all projector configs.